modules/denormalize_config.py

```diff
@@ -9,7 +9,7 @@
 
 
 def setup(builder):
-    config = builder.config
+    config = builder.config  # noqa
```

Comment:
I'm not actually sure what purpose this line serves; it is apparently useless, and we can probably cut it?